internal/meta/hcl_edit.go

```diff
@@ -38,6 +42,33 @@ func hclBlockAppendDependency(body *hclwrite.Body, deps []Dependency, cfgset map
 	return nil
 }
 
+func replaceIdAttrValuesWithTFAddr(body *hclwrite.Body, cfg ConfigInfo) bool {
+	resourceId := cfg.AzureResourceID.String()
```

Comment:
This should be the TF resource id instead of azure resource id

Reply:
Fixed, thx